config/site_settings.yml

```diff
@@ -911,6 +911,9 @@ files:
   s3_upload_bucket:
     default: ''
     regex: '^[a-z0-9\-\/]+$' # can't use '.' when using HTTPS
+  s3_endpoint:
+    default: 'https://s3.amazonaws.com'
+    regex: 'http.:\/\/+\w+\.\w+\.com' # http(s)://xyz.abc.com
```

Comment:
regex is wrong - copy the one from `s3_cdn_url` below

Reply:
Fixed, sorry about that! :+1: